src/azure-cli/azure/cli/command_modules/keyvault/tests/latest/test_keyvault_commands.py

```diff
@@ -107,6 +107,7 @@ def test_keyvault_mgmt(self, resource_group):
         # test policy set/delete
         self.cmd('keyvault set-policy -g {rg} -n {kv} --object-id {policy_id} --certificate-permissions get list',
                  checks=self.check('length(properties.accessPolicies[0].permissions.certificates)', 2))
+        self.cmd('keyvault set-policy -g {rg} -n {kv} --secret-permissions get', checks=self.check('name', '{kv}'))
```

Comment:
We can check the access policy like the previous test case.

Reply:
@jiasli I cannot check the access policy like the previous one because the `accessPolicies` is empty. Only if I specified the `--object-id` or `--spn` or `--upn`, I can check the access policies detail.
```
E           azure.cli.testsdk.exceptions.JMESPathCheckAssertionError: Query 'length(properties.accessPolicies[0].permissions.secrets)' doesn't yield expected value '1', instead the actual value is 'None'. Data:  
E           {
E             "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/cli_test_keyvault_mgmt000001/providers/Microsoft.KeyVault/vaults/cli-keyvault-000002",
E             "location": "westus",
E             "name": "cli-keyvault-000002",
E             "properties": {
E               "accessPolicies": [],
E               "createMode": null,
E               "enablePurgeProtection": true,
E               "enableSoftDelete": true,
E               "enabledForDeployment": true,
E               "enabledForDiskEncryption": true,
E               "enabledForTemplateDeployment": true,
E               "networkAcls": {
E                 "bypass": "AzureServices",
E                 "defaultAction": "Deny",
E                 "ipRules": [],
E                 "virtualNetworkRules": []
E               },
E               "provisioningState": "Succeeded",
E               "sku": {
E                 "name": "premium"
E               },
E               "tenantId": "54826b22-38d6-4fb2-bad9-b7b93a3e9c5a",
E               "vaultUri": "https://cli-keyvault-000002.vault.azure.net/"
E             },
E             "resourceGroup": "cli_test_keyvault_mgmt000001",
E             "tags": {},
E             "type": "Microsoft.KeyVault/vaults"
E           }
```

Reply:
I don't think so. If the response doesn't have `accessPolicies`, this should be a bug, either in the usage or code itself.

Reply:
@jiasli Fixed.